gtsfm/configs/deep_front_end_hilti.yaml

```diff
@@ -69,6 +69,8 @@ SceneOptimizer:
       output_reproj_error_thresh: 3 # for post-optimization filtering
       robust_measurement_noise: True
       shared_calib: False
+    
+    use_pose_slam_intitialization: True
```

Comment:
Can you please move it up with the target for multi-view plate? In both the configs?